wee_slack.py

```diff
@@ -5,6 +5,7 @@
 from collections import OrderedDict
 from functools import wraps
 from itertools import islice
+from itertools import count
```

Comment:
These two imports can be combined on one line.

Reply:
Done.